### PR TITLE
Add basic EVA support

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Addresses part of Issue #82 

This PR adds two things:
1 - modulefiles to load EVA on Orion and Hera
2 - a python script (and template file) to generate EVA YAML based off of JEDI YAML

To use the script:
`ush/eva/gen_eva_obs_yaml.py -i /path/to/work/dir/gdas_hofx.yaml -t ush/eva/jedi_gsi_compare.yaml -o /path/to/output/yamls/`

This will produce YAML files in the output directory using the template specified in `-t` for each observation space in the JEDI YAML file specified by `-i`.  The user can modify/create other template YAML files for different plot types as they see fit.

### CODE REVIEW

File: ush/eva/jedi_gsi_compare.yaml [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    # JEDI h(x) vs GSI h(x)
+    # ---------------------
+
+    - batch figure:
+        variables: *variables
+        @CHANNELSKEY@
+      figure:
+        layout: [1,1]
+        title: 'JEDI h(x) vs. GSI h(x) | @NAME@ @CYCLE@ | ${variable_title}'
+        output name: observation_scatter_plots/@CYCLE@/@NAME@/${variable}/@CHANNELVAR@/gsi_hofx_vs_jedi_hofx_@CYCLE@_@NAME@__${variable}@CHANNELVAR@.png
```
Comment (CatherineThomas-NOAA) [inline ush/eva/jedi_gsi_compare.yaml]: There's an extra underscore after NAME in the output filename

**Comment by CatherineThomas-NOAA** [comment]
Orion worked fine for me, but it's tripping up on Hera when trying to generate the map plots.  It can't connect to download a coastline file:
```
/scratch1/NCEPDEV/da/python/opt/core/miniconda3/4.6.14/envs/eva/lib/python3.7/site-packages/cartopy/io/__init__.py:241: DownloadWarning: Downloading: https://naciscdn.org/naturalearth/110m/physical/ne_110m_coastline.zip
  warnings.warn('Downloading: {}'.format(url), DownloadWarning)
...
urllib.error.URLError: <urlopen error [Errno 110] Connection timed out>
```

**Comment by CoryMartin-NOAA** [comment]
@CatherineThomas-NOAA yeah that's a hera problem if you've never plotted python maps before. It tries to download but cannot because of the firewall. Run `/scratch1/NCEPDEV/da/Cory.R.Martin/Python/ln_cartopy.sh` and then try it. That script links the files you need to the ones @kevindougherty-noaa has staged so that they don't need to download.

**Comment by CatherineThomas-NOAA** [comment]
@CoryMartin-NOAA  Worked like a charm. Thanks!

**Review by CatherineThomas-NOAA — APPROVED** [review]

**Pull request merged**